src/plugin.js

```diff
@@ -1,144 +1,60 @@
-import getConfig, { svgoJSONFilePath } from './getConfig'
-import path from '@skpm/path'
 import fs from '@skpm/fs'
 import dialog from '@skpm/dialog'
-import {spawnSync} from '@skpm/child_process'
-import svgo from 'svgo'
-import svgoPlugins from './svgo-plugins'
+import sketch from 'sketch'
 import UI from 'sketch/ui'
+import { optimize, loadConfig } from 'svgo'
+
 
 export function openSettings() {
```

Comment:
Maybe we can rename this now that the settings are gone:

```suggestion
export function showAbout() {
```

Reply:
Good catch!